app/models/models.scala

```diff
@@ -19,11 +20,8 @@ case class Author(
 
 object Author {
   def apply(authorData: AuthorFormData): Author = {
-    Author(
-      id = UUID.randomUUID().toString,
-      name = authorData.name,
-      url = authorData.url,
-      avatar = authorData.avatar
+    transform[AuthorFormData, Author](authorData,
+      "id" -> UUID.randomUUID().toString
```

Comment:
Am I right in thinking that this means Authors can never be edited and re-saved once written?